website/project/views/contributor.py

```diff
@@ -633,7 +633,6 @@ def claim_user_form(auth, **kwargs):
             # Redirect to CAS and authenticate the user with a verification key.
```

Comment:
> Just realized that the helper is for token. I do need new one for verification key. :) @icereval

Also, it goes without saying that any confirmation token that shows up in a URL will show up in server logs, so of course this token should be one-time use only. But I think it already is.

(technically, `math.log(2**160, 62) = 26.87180463312671`. But I'm all for 40 characters too; anyone trying to guess credentials is Not Our Friend, so let's not pretend to be nice to them.)